homework/src/index.js

```diff
@@ -33,14 +33,123 @@
+    showTableRepo(leftSection, data, 0);
+
+    select.addEventListener('change', (event) => {
+      document.getElementById('leftSection').innerHTML = "";
+      const theEvent = event.target.value;
+      showTableRepo(leftSection, data, theEvent);
+    });
+  }
+
+  
+
+  // drow the table
+function showTableRepo(leftSection, data, dataIndex) {
+  // create table of the contents
+  const lTable = createAndAppend('table', leftSection);
+  // line one
+  const tr1 = createAndAppend('tr', lTable);
+  createAndAppend('td', tr1, {html: 'Repository :', class: 'lable'});
+  const td1 = createAndAppend('td', tr1);
+  let nameLink = '<a href="' + data[dataIndex].html_url + '" target="_blank">' + data[dataIndex].name + '</a>';
+  createAndAppend('p', td1, {html: nameLink });
+  // line two
+  const tr2 = createAndAppend('tr', lTable);
+  createAndAppend('td', tr2, {html: 'Description :', class: 'lable'});
+  const td2 = createAndAppend('td', tr2);
+  createAndAppend('p', td2, {html: data[dataIndex].description});
+  // line three
+  const tr3 = createAndAppend('tr', lTable);
+  createAndAppend('td', tr3, {html: 'Forks :', class: 'lable'});
+  const td3 = createAndAppend('td', tr3);
+  createAndAppend('p', td3, {html: data[dataIndex].forks});
+  // line four
+  const tr4 = createAndAppend('tr', lTable);
+  createAndAppend('td', tr4, {html: 'Updated :', class: 'lable'});
+  const td4 = createAndAppend('td', tr4);
+  const defaultElDate = data[dataIndex].updated_at;
+  const defaultDate = new Date(defaultElDate);
+  const endDate = defaultDate.toLocaleString();
+  createAndAppend('p', td4, {html: endDate});
+}
+
+
+
+  // function who creates a right content section
+  function contentRight(container, data, select) {
+    const rightSection = createAndAppend('div', container, { id: 'rightSection', class: 'whiteframe' });
+    const ContributionsLable = createAndAppend('h5', rightSection, {class: 'ContributionsLable'});
+    ContributionsLable.innerHTML = "Contributions :";
+    let contrubutorsURL = data[0].contributors_url;
+    fetchJSON(contrubutorsURL, (err, subData) => {
+      if (err) {
+        createAndAppend('div', rightSection, { html: err.message, class: 'alert-error' });
+      } else {
+        showContributors(rightSection, subData);
+      }
+    });
+    select.addEventListener('change', (event) => {
+      document.getElementById('rightSection').innerHTML = "";
+      ContributionsLable.innerHTML = "Contributions :";
+      const theEvent = event.target.value;
+      let contrubutorsURL = data[theEvent].contributors_url;
+      fetchJSON(contrubutorsURL, (err, subData) => {
+        if (err) {
+          createAndAppend('div', rightSection, { html: err.message, class: 'alert-error' });
+        } else {
+          showContributors(rightSection, subData);
+        }
+      });
+    });
+  }
+
+
+
+
+/*  ************************************ */
+function showContributors(rightSection, subData) {
+  subData.forEach(el => {
```

Comment:
Your code would be far more clear if you used variable names that accurately describe what they contain:

```js
contributors.forEach(contributor => {
```